Tasks/src/Calculator.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+        public static void main (String[] args) {
+            double sum, num1, num2;
+            Scanner scd = new Scanner(System.in);
```

Comment:
You don't need 2 variables for Scanner (scd, scs).

Reply:
Ok, will be fixed in next PR.